panda/src/bullet/bulletRigidBodyNode.cxx

```diff
@@ -47,6 +47,28 @@ BulletRigidBodyNode(const char *name) : BulletBodyNode(name) {
 }
 
 /**
+ * Do not call the copy constructor directly; instead, use make_copy() or
+ * copy_subgraph() to make a copy of a node.
+ */
+BulletRigidBodyNode::
+BulletRigidBodyNode(const BulletRigidBodyNode &copy) :
+  BulletBodyNode(copy)
+{
+  _motion = new MotionState(*copy._motion);
+  _rigid = new btRigidBody(*copy._rigid);
```

Comment:
You need to call `setUserPointer` on `_rigid` since it otherwise still points to the first object.